e2e-tests/package.json

```diff
@@ -40,5 +44,16 @@
     "del": "^8.0.0",
     "minimist": "^1.2.8",
     "mocha-junit-reporter": "^2.2.1"
+  },
+  "nyc": {
+    "all": true,
```

Comment:
This is probably not needed, because running `nyc instrument` by default should be instrumenting all matched source code. 
Additionally, if you narrow down the scope with the include `packages/legacy-workbench/src/**/*.js`, then you probably don't need the exclude configuration.